packages/frontend-shared/src/locales/en-US.json

```
@@ -110,7 +110,7 @@
       },
       "importEmptySpec": {
         "header": "Create a new spec",
-        "description": "We'll generate an empty spec file to start testing components.",
+        "description": "We'll generate a starter spec file to start testing components.",
```

```suggestion
        "description": "We'll generate a template spec file which can be used to start testing your components.",
```
The verbiage between component & e2e testing ideally will be pretty similar.

---

Same comment as [here](https://github.com/cypress-io/cypress/pull/24825/files/cb11c3c3d520451f71dc5aadb59068dd41105f3a#r1056628681).